doc/source/quickstart.rst

```diff
@@ -0,0 +1,422 @@
+KIWI Quickstart
+===============
+
```

Comment:
So we call this chapter Quickstart and within the chapter we have a topic also named Quickstart, isn't that confusing ? I think there is much more information in here than just a quick start. It feels more like an introduction. Thoughts ?